packages/smithy-http/src/smithy_http/serializers.py

```diff
@@ -486,7 +486,7 @@ def write_big_decimal(self, schema: Schema, value: Decimal) -> None:
 
     def write_string(self, schema: Schema, value: str) -> None:
         key = self._key or schema.expect_trait(HTTPQueryTrait).key
-        self.query_params.append((key, urlquote(value, safe="")))
```

Comment:
Does this have any implications on our current signing setup? The signers can double encode but I'm wondering if we're setting that appropriately if we're removing this.